llvm/test/Transforms/LoopVectorize/outer_loop_test1.ll

```diff
@@ -13,7 +13,7 @@
 ;   }
 ; }
 ;
-; RUN: opt -S -passes=loop-vectorize -enable-vplan-native-path -verify-loop-info < %s | FileCheck %s
+; RUN: opt -S -passes=loop-vectorize -enable-vplan-native-path -verify-loop-info -verify-dom-info < %s | FileCheck %s
```

Comment:
Should that be added once DT is updated for native?

Reply:
Removed, thanks!